Index.html

```diff
@@ -38,7 +38,136 @@ <h2 class="greenheader">
             <li><a href="https://github.com/CindyDorantes"><img src="./Images/github.png" alt="GitHub icon" class="socialicon"></a></li>
             <li><a href="https://angel.co/"><img src="./Images/angellist.png" alt="AngelList icon" class="socialicon"></a></li>
         </ul>
-        
     </section>
+        <p><img src="./Images/arrow_button.png" alt="arrow button" class="arrow_button"></p>
+    <!--here begins the 'Works' section-->
+    <section class="works">
+        <h2 class="works-heading">My recent Works <br> <img src="./Images/Indicator.png" alt="green line divisor"></h2>
+        <br>
+        <!--First work-item-->
+        <div class="work-item01">
+            <div class="work-image"></div>
+            <div class="work-content">
+                <h3 class="work-title">Multi-Post Stories Gain+Glory</h3>
+                <ul class="cards">
+                    <li>Ruby on rails</li>
+                    <li>CSS</li>
+                    <li>JavScript</li>
+                    <li>HTML</li>
+                </ul>                
+                <div class="see-project">See project</div>
+            </div>
```

Comment:
- [ **OPTIONAL** ] For good HTML semantics, I suggest you use the appropriate to make your code more professional. This is supposed to be used as a `<button>` tag and not a `div`. I suggest you change it and apply it on all `See Project` buttons as the previous reviewer forgot to point it out.
- [ **OPTIONAL** ] Kindly make sure all your buttons, all `See Project` and `Get my resume` buttons, follow the button styling: `Active`, `Hover` and `Pressed` as for the Figma template. You can check the image below from Figma to get the appropriate colors.
![buttons_color](https://user-images.githubusercontent.com/83042742/172021127-9aa07628-d99e-433a-857a-4f850748b450.png)